core/oslib/audiobackend_alsa.cpp

```diff
@@ -175,12 +175,81 @@ static void alsa_term()
 	snd_pcm_close(handle);
 }
 
+std::vector<std::string> alsa_get_devicelist()
```

Comment:
I think you can just use `vector<string>` here to avoid cramping your hands. Afair reicast has `<vector>`, `<string>` and `using std;` on the main include file (`types.h`)

Reply:
I think I will leave this as is, as I already typed this. :-)
I will remember this for the future and omit the ``std::``.

Reply:
well, to keep in mind for the future

Reply:
Yep, I will.